spec/bullet/rack_spec.rb

```diff
@@ -60,15 +60,15 @@ module Bullet
         it "should return original response body" do
           expected_response = Support::ResponseDouble.new "Actual body"
           app.response = expected_response
-          _, _, response = middleware.call([])
+          _, _, response = middleware.call({})
```

Comment:
I think it should be a hash because Rack specification says followings.

> The environment must be an instance of Hash

http://www.rubydoc.info/github/rack/rack/file/SPEC